platform/admin/core/admin-core-impl/src/test/java/org/codice/ddf/admin/core/impl/AdminConsoleServiceTest.java

```diff
@@ -811,6 +811,28 @@ public void testUpdate() throws Exception {
   }
 
   @Test
+  public void testSanitizeUIConfiguration() throws Exception {
+    AdminConsoleService configAdmin = getConfigAdmin();
+
+    // Initialize sanitized values.
+    Map<String, Object> currentProps = new Hashtable<>();
+    currentProps.put(
+        "color",
+        "white; height: 210px; z-index: 999999; background-image: url(https://gifimage.net/wp-content/uploads/2017/06/troll-face-gif-8.gif); background-repeat: no-repeat; background-attachment: fixed; background-position-x: center");
```

Comment:
I hate to be a fun-killer here, but hardcoding public URLs that aren't connexta-owned in our tests probably isn't the best idea. 😢  Since we're just making sure that semicolons aren't allowed, any string value with semicolons in it would work for testing.